pymanopt/manifolds/psd.py

```diff
@@ -79,7 +79,9 @@ def retr(self, X, G):
         return self.exp(X, G)
 
     def norm(self, x, u):
-        return la.norm(la.solve(x, u))
+        c = np.linalg.cholesky(x)
+        c_inv = np.linalg.inv(c)
```

Comment:
It's usually bad practice to explicitly invert a matrix. Since you only need to apply the inverse to certain matrices, it should be possible to call a linear system solver directly (and hopefully you can tell the solver that c is triangular, if it indeed is -- I'm not familiar with np.linalg).

Reply:
You are absolutely right. I did some tests with `scipy.linalg.solve_triangular`, which is (in that case) as fast as cholesky+inv, but I ran into the same problem as the regular np.linalg.solve : the result is not reliable as the results are not always positive definite. I also tried `scipy.linalg.sqrtm` and the computation of c_inv via eigenvalue decomposition, which are both slower than cholesky+inv.

Reply:
Understood, thanks for these tests -- can you add a short comment in the code to that effect? This way, other people who wonder about this will be informed too. Thanks!